src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditContactDetails.cshtml

```diff
@@ -41,7 +41,7 @@
         <div class="govuk-body">
             Provide the name and address of the person who is the primary contact. We will send all correspondence to this contact.
             <br /><br />
```

Comment:
@stevelee-civica Not related to this change, but the <br /><br /> tags can you remove and put the two sentances in their own p tags with class = govuk-body as the screen reader will moan about this

Reply:
@stevelee-civica  this comment